src/app/components.jsx

```diff
@@ -118,6 +119,15 @@ const components = [
       onSelect: () => {},
     },
   },
+  {
+    name: 'Search Input',
+    component: SearchInput,
+    function: 'blah',
```

Comment:
Maybe something like:

Function: Filter data displayed on the screen (autocomplete, list of fields etc..)
Purpose: Provide the user with an indication that typing something in the input box will search something